scripts/photo.js

```diff
@@ -74,15 +74,27 @@ photo.load = function(photoID, albumID) {
 };
 
 photo.update_display_overlay = function () {
+	let showDesc = false; // store the intial value
+
 	lychee.image_overlay = !lychee.image_overlay;
-	if(!lychee.image_overlay)
-	{
+	if(!lychee.image_overlay) {
 		$('#image_overlay').remove();
+	} else {
+		// do this while a photo is showing
+		setInterval(() => {
+			$('#image_overlay').remove();
+			if(showDesc) {
+				lychee.imageview.append(build.overlay_image(photo.json.description))
```

Comment:
this line looks weird.
You are will be printing  `photo.json.description.description` cf the other file: `<div id="image_overlay"><p>${ data.description }</p></div>`